### PR TITLE
update views (button, align, h2)

### PR DESCRIPTION
**Pull request by JonasJallet**

### CODE REVIEW

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflicts with dev

**Comment by JonasJallet** [comment]
update with dev merge on my branch

File: src/Model/ArtistManager.php [hunk 1]
```diff
@@ -35,4 +35,13 @@ public function update(array $artist): bool
         $statement->bindValue(':image', $artist['image'], PDO::PARAM_STR);
         return $statement->execute();
     }
+
+    public function selectRandomArtists(): array
+    {
+        $query = "SELECT name, style, image FROM " . static::TABLE . " ORDER BY RAND() LIMIT 4";
+        $statement = $this->pdo->query($query);
+        $allTables = $statement->fetchAll();
```
Comment (aldaitzwild) [inline src/Model/ArtistManager.php]: It's stil not a decent name for this variable :-(

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Model/ArtistManager.php (hunk 1 above)
Comment (JonasJallet) [reply src/Model/ArtistManager.php]: sorry i didn't see this error previously, i hope don't have misunderstood your request

File: src/Model/ConcertManager.php [hunk 2]
```diff
@@ -60,8 +60,8 @@ public function selectAllArtists(): array
     {
         $query = "SELECT * FROM artist";
         $statement = $this->pdo->query($query);
-        $allArtists = $statement->fetchAll();
+        $allTables = $statement->fetchAll();
```
Comment (aldaitzwild) [inline src/Model/ConcertManager.php]: ok it's getting weird ... Let's say "allTables" is never a good name for a variable, EXECPT if you work at IKEA and you have to present a page with ... yeah all the Tables

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Model/ConcertManager.php (hunk 2 above)
Comment (JonasJallet) [reply src/Model/ConcertManager.php]: maybe i gonna be better for return working at Ikea, i hope it's good now

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**